src/de.hpi.swa.graal.squeak/src/de/hpi/swa/graal/squeak/nodes/ExecuteContextNode.java

```diff
@@ -210,6 +265,14 @@ private Object startBytecode(final VirtualFrame frame) {
                 pc = node.getSuccessorIndex();
                 if (node instanceof AbstractSendNode) {
                     FrameAccess.setInstructionPointer(frame, code, pc);
+                } else if (node instanceof SistaCallInlinePrimitiveNode) {
+                    scipProfile.enter();
```

Comment:
I am new to truffle, so this may not do what I think it does
I intended to signal the compiler that this is an unlikely branch (as of now completely unused, as you pointed out)

Reply:
No problem at all, happy to help :)